### PR TITLE
Bumping versions to 0.15.0

### PR DESCRIPTION
**Pull request by kylebshr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -378,10 +386,11 @@ Box(cornerStyle: .capsule)
...
 
-[main]: https://github.com/square/Blueprint/compare/0.14.0...HEAD
-[0.14.0]: https://github.com/square/Blueprint/compare/0.14.0...0.13.1
-[0.13.1]: https://github.com/square/Blueprint/compare/0.13.1...0.13.0
-[0.13.0]: https://github.com/square/Blueprint/compare/0.13.0...0.12.2
+[main]: https://github.com/square/Blueprint/compare/0.15.0...HEAD
+[0.15.0]: https://github.com/square/Blueprint/compare/0.14.0...0.15.0
+[0.14.0]: https://github.com/square/Blueprint/compare/0.13.1...0.14.0
+[0.13.1]: https://github.com/square/Blueprint/compare/0.13.0...0.13.1
+[0.13.0]: https://github.com/square/Blueprint/compare/0.12.2...0.13.0
```
Comment (kylebshr) [inline CHANGELOG.md:393]: These started being backwards for some reason.. not sure they don't work, but I assume we want it to be consistent

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,24 +45,14 @@ into
 Box(cornerStyle: .capsule)
 ```
 
-- Add `accessibilityFrameSize` to `AccessibilityElement` for manually specifying a size for the frame rendered by Voice Over.
+- Add `accessibilityFrameSize` to `AccessibilityElement` for manually specifying a size for the frame rendered by Voice Over. ([#144])
 
-### Removed
+- Add `Opacity` element for modifying the opacity of a wrapped element. ([#147])
```
Comment (bencochran) [inline CHANGELOG.md:50]: Thanks! Totally forgot this in my PR

Comment (kylebshr) [reply CHANGELOG.md:50]: haha nw! I do all the time (and also forgot to ask for it in the code review)

**Review by bencochran — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (watt) [reply CHANGELOG.md:393]: Oops, that was probably me!

**Pull request merged**